v03_pipeline/lib/reference_datasets/reference_dataset.py

```diff
@@ -0,0 +1,93 @@
+from enum import Enum
+from types import MappingProxyType
+
+from v03_pipeline.lib.model import AccessControl, DatasetType, Env, ReferenceGenome
+from v03_pipeline.lib.reference_datasets.cadd import load_cadd_ht_from_raw_dataset
+from v03_pipeline.lib.reference_datasets.hgmd import (
+    HGMD_ENUM_SELECT,
+    download_and_import_hgmd_vcf,
+)
+
+LOAD_DATASET_FUNC = 'load_parsed_dataset_func'
+VERSION = 'version'
+RAW_DATASET_PATH = 'raw_dataset_path'
+ENUM_SELECT = 'enum_select'
+
+
+class ReferenceDataset(str, Enum):
+    cadd = 'cadd'
+    hgmd = 'hgmd'
+
+    CONFIG = MappingProxyType(
+        {
+            'cadd': {
+                LOAD_DATASET_FUNC: load_cadd_ht_from_raw_dataset,
+                ReferenceGenome.GRCh37: {
+                    VERSION: '1.0',
+                    RAW_DATASET_PATH: [
+                        'https://krishna.gs.washington.edu/download/CADD/v1.7/GRCh37/whole_genome_SNVs.tsv.gz',
```

Comment:
do we have any concerns about updating from CADD 1.6 to 1.7?

Reply:
Lynn says we are good!

Reply:
good question, currently checking out the release notes: https://cadd.bihealth.org/static/ReleaseNotes_CADD_v1.7.pdf

Reply:
sounds good. I still need to test/run the new cadd script with the 1.7 data sources.